README.md

```diff
@@ -22,6 +22,49 @@ $ git clone https://github.com/foundersandcoders/morning-challenge-traffic-light
 $ npm i && npm run live
 ```
 
+## What is first-class function?
+
+First class function is similar to being a first class citizen - it has the right to freely move across boundaries in the country without being questioned of its purpose. In other words, it is an entity which supports all the operations generally available to other entities
```

Comment:
Personally I don't like this metaphor, I don't think it makes what a first class function is any clearer, and sadly some of our students at FAC aren't lucky enough to experience freedom of movement so I'd prefer if we could remove it?